main.js

```diff
@@ -42,7 +45,11 @@ const createHardresetHandler = (eXecutable, hardResetMethod, argv) =>
 
 module.exports = (glob, options = {}) => {
   const browserWindows = []
-  const watcher = chokidar.watch(glob, Object.assign({ ignored: [ignoredPaths, mainFile] }, options))
+
+  const ignored = [ignoredPaths, mainFile].filter(Boolean)
+  const watcherDirectory = path.resolve(appPath, glob)
+
+  const watcher = chokidar.watch(watcherDirectory, Object.assign({ ignored }, options))
```

Comment:
Changes above would render hard resets impossible!

Reply:
Ok, I understand. Thank you for your comment.